lib/project_types/theme/commands/generate/env.rb

```diff
@@ -0,0 +1,78 @@
+module Theme
+  module Commands
+    class Generate
+      class Env < ShopifyCli::SubCommand
+        options do |parser, flags|
+          parser.on('--store=STORE') { |url| flags[:store] = url }
+          parser.on('--password=PASSWORD') { |p| flags[:password] = p }
+          parser.on('--themeid=THEME_ID') { |id| flags[:themeid] = id }
+          parser.on('--env=ENV') { |env| flags[:env] = env }
+        end
+
+        def call(*)
+          default_store, default_password = fetch_credentials
+          store = ask_store(default_store) || default_store
+          password = ask_password(default_password) || default_password
+          themeid = ask_theme(store: store, password: password)
+          env = options.flags[:env]
+
+          Themekit.generate_env(@ctx, store: store, password: password, themeid: themeid, env: env)
+        end
+
+        def self.help
+          ShopifyCli::Context.message('theme.generate.env.help', ShopifyCli::TOOL_NAME)
+        end
+
+        private
+
+        def fetch_credentials
+          begin
+            config = YAML.load_file('config.yml')
+            store = config['development']['store']
+            password = config['development']['password']
+          rescue
+            # what happens if no config
```

Comment:
I actually forgot to take out this comment.

Currently, the behaviour is that if there is no config to take the default from, it does not allow an empty field. It is, however, a good idea to check for existence rather than swallowing the error.